src/Servers/Kestrel/Transport.MsQuic/src/Internal/SendContext.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Buffers;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using static Microsoft.AspNetCore.Server.Kestrel.Transport.MsQuic.Internal.MsQuicNativeMethods;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Transport.MsQuic.Internal
+{
+    public sealed class SendContext
```

Comment:
Eventually we want this to be a SafeHandle, right?

Reply:
Yeah, too big of a change for this PR.